browser/brave_shields/ad_block_service_browsertest.cc

```diff
@@ -35,6 +37,7 @@
 #include "content/public/test/browser_test_utils.h"
 #include "extensions/test/extension_test_message_listener.h"
 #include "net/dns/mock_host_resolver.h"
+#include "services/network/host_resolver.h"
```

Comment:
don't we also need to add `//services/network:network_service` to the deps in browser tests section of `//brave/test/BUILD.gn`?

Reply:
@darkdh Good call, adding now.

Should I add it to `browser/net/BUILD.gn` too, or is the `browser/DEPS` update sufficient?

Reply:
oh, yes please. `DEPS` is only for rule enforcing whether you can include certain headers so you still need to add proper `deps` in BUILD.gn otherwise, you might get undefined references under  certain circumstances.

Reply:
gotcha, updated!